app/components/Explorer/Assets.jsx

```diff
@@ -36,6 +41,7 @@ class Assets extends React.Component {
             nextState.filterMPA !== this.state.filterMPA ||
             nextState.filterUIA !== this.state.filterUIA ||
             nextState.filterPM !== this.state.filterPM ||
+            nextProps.isLoading !== this.state.isLoading ||
```

Comment:
This should be `nextState.isLoading` no?

Either way the `!utils.are_equal_shallow(nextState, this.state)` on the next line handles all state comparisons in this case, so we can actually remove lines 41 through 44

Reply:
That's entirely correct. I've made a typo, and didn't notice cause the are_equal_shallow does the work 👍